FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -153,12 +153,16 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
                     }
                 }
 
-                _result.CompleteTime = DateTimeOffset.UtcNow;
+                var processEndtime = DateTimeOffset.UtcNow;
+                _result.CompleteTime = processEndtime;
 
                 progress.Report(JsonConvert.SerializeObject(_result));
 
-                _diagnosticLogger.LogInformation("Finish FhirToDataLake job.");
-                _logger.LogInformation($"Finish FhirToDataLake orchestrator job {_jobInfo.Id}");
+                var jobLatency = processEndtime - _inputData.DataEndTime;
+                _metricsLogger.LogProcessLatencyMetric(jobLatency.TotalSeconds);
```

Comment:
seem the job latency is only meaningful for the incremental job?

Reply:
Yes. Currently it's coarse-grained to report latency at trigger level.